example/testcontent/models.py

```diff
@@ -109,3 +109,8 @@ def get_absolute_url(self):
 
 class TestLiveBlog(Content, AbstractLiveBlog):
     """Example LiveBlog implementation"""
+
+    class Mapping(Content.Mapping):
```

Comment:
I suspect this mapping needs to be added to each concrete implementation (AVC, ON, etc) b/c DJES doesn't support multiple base class Mappings?